librisworks/src/main/groovy/se/kb/libris/mergeworks/Util.groovy

```diff
@@ -78,20 +82,30 @@ class Util {
         hasTitle.any { it['mainTitle'] && normalize((String) it['mainTitle']) in GENERIC_TITLES }
     }
 
-    static List dropSubtitles(List hasTitle) {
+    static List dropGenericSubTitles(List hasTitle) {
         hasTitle.collect {
             def copy = new TreeMap(it)
-            DocumentUtil.traverse(copy) { value, path ->
-                if (('subtitle' in path || 'titleRemainder' in path) && value instanceof String) {
-                    new DocumentUtil.Remove()
+            if (copy['subtitle'] || copy['titleRemainder']) {
+                DocumentUtil.traverse(copy) { value, path ->
+                    if (('subtitle' in path || 'titleRemainder' in path) && value instanceof String) {
+                        if (genericSubtitle(value)) {
+                            new DocumentUtil.Remove()
+                        } else {
+                            ((List) value.split(':')).with {
+                                if (it.size() > 1 && genericSubtitle(it.last().trim())) {
+                                    new DocumentUtil.Replace(value.replaceFirst(~/\s*:.+$/, ''))
+                                }
```

Comment:
Can you add a a comment/example explaining why we do this?

Reply:
See [c86d9ee](https://github.com/libris/librisxl/pull/1325/commits/c86d9ee16d438f059bcf4449ad18bfe9ca8b5b50).
When there's time I'll go through the whole module and document more of the code.